plugins/itemswitheffects/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "author": "PoisonF5god",
+    "version":"1.0.0",
+    "name": "ItemsWithEffectsPreset",
+    "description": "adds presets for items that give off Mob Effects.",
+    "id": "4edf8fd3-45b5-424d-a433-0ea561ce63d0",
+    "target": "both",
```

Comment:
This extension only works on v2 so please update the target to "v2" as well

Reply:
Will do does everything else look good?
-PoisonF5god